### DISCUSSION TITLE
Specific pweather permissions.

### DISCUSSION THREAD

**Discussion by hirata-estate** [EssentialsX/Essentials#4525] [Ideas and Feature Suggestions] [answered]
There's a bug with **'/pweather storm'** that makes tridents with Riptide III still work even though it's supposed to be server side only. Reference: [#2126](https://github.com/EssentialsX/Essentials/issues/2126). From what I understood, this isn't going to be fixed as it isn't a essentials issue. (bug still works on 1.17.1, essentials version: 2.19.1-dev+4-fde6524.)

Since it isn't going to be fixed, can a permission for players to only have access to **'/pweather clear/sun'**, or not let them use **'/pweather storm?'** This way we can still give pweather as a rank perk without them being able to use this exploit.

Thank you!

**Comment by JRoy** [comment, marked as answer]
This can be safely ignored
